docs/content/configuration/v1alpha1-reference.md

```diff
@@ -0,0 +1,303 @@
+---
+title: v1alpha1 Reference
+date: 2019-10-04T19:40:55-07:00
+draft: false
+weight: 20
+menu:
+  docs:
+    parent: 'configuration'
+---
+
+Talos User Data is responsible for the host and Kubernetes configuration, and it is independent of other cloud config / cloud init data.
+
+## Differences from v0
+
+The main driver in introducing a new configuration file format is to reduce the complexity and make it more approachable.
+The redesign proposal can be found [here](https://github.com/talos-systems/talos/blob/master/docs/proposals/20190708-MachineConfig.md).
+The major change between these two versions is the introduction of `machine` and `cluster` configuration objects.
+Machine configuration data deals with the configuration of the host itself whereas cluster configuration data deals with the configuration of the clustering on top of Talos ( ex, Kubernetes configuration ).
+
+## Version
+
+``Version`` represents the Talos userdata configuration version.
+
+This denotes what the schema of the configuration file.
+
+```yaml
+version: v1alpha1
+```
+
+## Machine Configuration
+
+```yaml
+machine:
+  type: string
+  token: string
+  ca:
+    crt: string ( base64 encoded certificate )
+    key: string ( base64 encoded key )
+  kubelet: (optional)
+    image: string
+    extraArgs: []string
+  network: (optional)
+    hostname: string
+    interfaces:
+      - interface: string
+        cidr: string
+        dhcp: bool
+        ignore: bool
+  install: (optional)
+    disk: string
+    extraKernelArgs: []string
+    image: string
+    bootloader: bool
+    wipe: bool
+    force: bool
+```
+
+### machine.type
+
+``type`` defines the type/role of a node.
+
+Acceptable values are:
+-``init``
+-``controlplane``
+-``worker``
+
+#### Init
+
+Init node type designates the first control plane node to come up.
+You can think of it like a bootstrap node.
```

Comment:
Note for next config version: I think, eventually, it makes send to use `bootstrap` instead if `init`.